awaitwhat/utils/helpers.py

```diff
@@ -0,0 +1,15 @@
+import asyncio
+import signal
+import sys
+
+from .. import dot
```

Comment:
Instead of doing relative import (`..`), it's better to import it explicitly with `from awaitwhat import dot`.